server/routes/api/conversations.js

```diff
@@ -78,4 +77,19 @@ router.get("/", async (req, res, next) => {
   }
 });
 
-module.exports = router;
+
+router.post("/resetUnreadMsgForCurrentUser" ,async (req, res, next) => {
+  try {
+    if (!req.user) {
+      return res.sendStatus(401);
+    }
+    const { id, user1UnreadMsg, user2UnreadMsg } = req.body;
+    const conversation = await Conversation.updateConversation(id, user1UnreadMsg, user2UnreadMsg);
+    res.json(conversation);
```

Comment:
Nice work protecting the route against unauthenticated users. We also want to protect the route against unauthorized users (for example, users who are not part of the conversation). 
As this route stands at the moment, as long as I have a user, if I manage to acquire the conversation id of a different conversation, then I'll be able to mark it as unread using a post request in Postman, even if I am not part of the conversation.